docs/docs/Integrations/integrations-mcp.md

```diff
@@ -0,0 +1,113 @@
+---
+title: MCP (Model context protocol)
+slug: /integrations-mcp
+---
+
+# Integrate Langflow with MCP
+
+Langflow integrates with the [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction). This allows you to use your Langflow flows as tools in other applications that support the MCP protocol, or extend Langflow with the [MCP stdio component](/components-tools#mcp-tools-stdio) to access MCP servers.
+
+* Access all of your flows as tools from Claude Desktop
+* Connect a [Datastax Astra DB MCP server](https://github.com/datastax/astra-db-mcp) to Cursor
+* Connect a [Datastax Astra DB MCP server](https://github.com/datastax/astra-db-mcp) to Langflow
+
+This guide will show you how to use Langflow as an MCP server with Claude Desktop as the client.
+
+## Access all of your flows as tools from Claude Desktop
+
+The MCP server configuration is added to Claude, which can then access all of your flows as tools with the MCP protocol.
+
+### Prerequisites
+
+* [Claude desktop](https://claude.ai/download) installed
+* [uv installed](https://docs.astral.sh/uv/getting-started/installation/) to run uvx commands
+
+### Add Langflow as an MCP server to Claude
+
+1. Create at least one flow, and note your host, for example, `http://127.0.0.1:7863`.
+
+2. Open Claude Desktop. Go to **Settings** > **Developer** > **Edit Config**.
```

Comment:
Avoid documenting the steps for a 3rd-party app. Provide general guidance and link to that product's docs.